### DISCUSSION TITLE
Issues with ACE 3.16.0, missing compat files, server icon appears red

### DISCUSSION THREAD

**Discussion by celticalliance** [acemod/ACE3#9530] [General]
Hello,

I have just updated ACE and CBA to 3.16.0, and for some reason the server icons now appear red in the server browser, while I have everything installed correctly on server and client-level. And never had any issues with the previous versions.

I can connect to our servers without issues. I however wonder where the compat files have gone, since we are using RKSL Attachments, and always used the compat_rksl_pm_ii file.

Can someone please explain to me what I am missing, and how to obtain it? I can not afford for our servers to appear with red icons as we have an Operation on the way. Are the server keys provided with the install correct? There used to be a random number behind it but not anymore.

Thank you for your help.

**Comment by jonpas** [comment]
Stop opening duplicates, you already have a discussion open.
